www.foia.gov/_includes/legacy/popup.html

```diff
@@ -11,3 +11,23 @@
        </div>
 </div>
 <!-- End Popup Window -->
+<!-- Start Popup Window -->
```

Comment:
Should we add a note about which pages use one popup vs. the other? The video link is clearly its own thing, but these two are confusing together.